resources/views/components/nav.blade.php

```diff
@@ -67,7 +67,7 @@
                                 <li>{!! link_to_route("site.atc.bookings", "Bookings") !!}</li>
                             </ul>
                         </li>
-                        @if(currentUserHasAuth() && Auth::user()->qualificationAtc->isS1)
+                        @if(currentUserHasAuth() && Auth::user()->qualificationAtc && Auth::user()->qualificationAtc->isS1)
```

Comment:
This menu option shows for all to avoid confusion and a redirect is present in the controller to handle non-S1s trying to get in

Reply:
https://github.com/VATSIM-UK/core/blob/master/resources/views/components/nav.blade.php#L70 this is already present on prod, I simply added this to stop it error-ing some tests

Reply:
There should never be an instance where we have someone with auth and without qualifications. Which tests were erroring?

Reply:
Uh my tests use a user without quals ;)

Reply:
The `Account::class` factory should spin up a state and qualification ideally

Reply:
Do we want to remove that restriction on the nav aswell?

Reply:
Yeah we can do and just handle it in the controller